ui/pages/settings/snaps-list-tab/view-snap/index.scss

```diff
@@ -0,0 +1,30 @@
+div .view-snap-subheader {
+    @media screen and (max-width: $break-small) {
+        display: flex;
+        margin-right: 0px;
+        padding-right: 24px;
+        padding-left: 24px;
```

Comment:
`padding-inline-start` and `padding-inline-end` would be better for RTL; if there's no padding top or left, we can do `padding: 0 24px;`